pkgs/top-level/release.nix

```diff
@@ -22,7 +22,7 @@
   supportedSystems ? builtins.fromJSON (builtins.readFile ../../ci/supportedSystems.json),
   # The platform triples for which we build bootstrap tools.
   bootstrapConfigs ? [
-    "aarch64-apple-darwin"
+    "arm64-apple-darwin"
```

Comment:
has the triple we use for aarch4 darwin been inconsistent between pkgs vs bootstrap for ages? :c

`aarch64-darwin`'s `pkgs.hostPlatform.config` does give `arm64-apple-darwin`

Reply:
So I've ended up centralizing what we know here https://github.com/NixOS/nixpkgs/pull/444862#issuecomment-3366730973

Reply:
It seems like we're using `darwinArch` to turn `aarch64` -> `arm64` in some places, maybe we can clean those up in a followup if they were only needed due to this inconsistency.

```nix
  darwinArch = cpu: if cpu.name == "aarch64" then "arm64" else cpu.name;
```